test/test.kafkaClient.js

```diff
@@ -806,4 +806,72 @@ describe('Kafka Client', function () {
       });
     });
   });
+
+  describe('#createTopics', function () {
+    let sandbox, client;
+
+    beforeEach(function (done) {
+      if (process.env.KAFKA_VERSION === '0.9') {
+        this.skip();
+      }
+
+      sandbox = sinon.sandbox.create();
+      client = new Client({
+        kafkaHost: 'localhost:9092'
+      });
+      client.once('ready', done);
+    });
+
+    afterEach(function (done) {
+      sandbox.restore();
+      client.close(done);
+    });
+
+    it('should create given topics', function (done) {
+      const topic1 = uuid.v4();
+      const topic1ReplicationFactor = 1;
+      const topic1Partitions = 5;
+      const topic2 = uuid.v4();
+      const topic2ReplicationFactor = 1;
+      const topic2Partitions = 1;
+
+      client.createTopics([
+        {
+          topic: topic1,
+          partitions: topic1Partitions,
+          replicationFactor: topic1ReplicationFactor
+        },
+        {
+          topic: topic2,
+          partitions: topic2Partitions,
+          replicationFactor: topic2ReplicationFactor
+        }
+      ], (error, result) => {
+        should.not.exist(error);
+        result.should.be.empty;
```

Comment:
can we also fetch the topic metadata and verify the partitions/replication factor for newly created topics?

Reply:
Good point - added.